scripts/dl_cleanup.py

```diff
@@ -14,7 +14,7 @@
 import re
 import getopt
 
-# Commandline options
+# Command-line options
```

Comment:
This is the generally preferred spelling for the term. I'm of course fine w/ dropping any individual thing, but the term only appeared in comments as opposed to code, so it seemed harmless to correct it.

Reply:
> This is the generally preferred spelling for the term. 

Who says that? My research yields "command line" ...

Reply:
I'd be happy to take either `command line` or `command-line` over `commandline`.

I know that wikipedia is not a source, but fwiw, the article is relatively (not absolutely) consistent in using the hyphenated form:
https://en.wikipedia.org/wiki/Command-line_interface

Here's the MSDN Visual Studio doc https://docs.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2019 which is similarly relatively consistent.

And a similar doc for VS Code (which admittedly is related to VS) https://code.visualstudio.com/docs/editor/command-line

Some content is definitely mixed https://www.geeksforgeeks.org/difference-between-terminal-console-shell-and-command-line/ appears to be close to 50/50.

I think roughly when people are writing more formally they tend to use command-line, and when generic people are asking or quickly responding, they tend to not realize that formally it should be hyphenated.

Note: I'm quite happy to adopt whichever spelling a project selects. I'm not trying to evangelize any particular spelling.